### ISSUE TITLE
[4.x] Bard set picker positioned incorrectly in Safari

### ISSUE THREAD

**Issue by robdekort** [statamic/cms#7759]
### Bug description

In v4 (8f4fe9e) the new set picker in Bard is positioned in correctly in Safari. This is a Bard in a replicator but it happens with Bard at root level as well.

The replicator picker works as intended. 

<img width="1565" alt="Screenshot 2023-03-24 at 09 05 58" src="https://user-images.githubusercontent.com/69107412/227461436-5aeade16-1434-465b-861d-7543fac5dd48.png">

### How to reproduce

Test the bard picker in a replicator.

### Environment

```yaml
Environment
Application Name: V4
Laravel Version: 9.52.4
PHP Version: 8.2.3
Composer Version: 2.2.7
Environment: local
Debug Mode: ENABLED
URL: v4.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 4
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: dev-master PRO

Statamic Addons
studio1902/statamic-peak-browser-appearance: 1.0
studio1902/statamic-peak-commands: 1.26
studio1902/statamic-peak-seo: 1.7
studio1902/statamic-peak-tools: 2.2
```

### Installation

Starter Kit using via CLI

### Antlers Parser

runtime (new)

**Comment by jasonvarga** [comment]
Ah thanks, this is weird.

While I was developing that (using Chrome) I noticed that when I autofocused the search field in the picker, it would scroll to the top of the page.

I worked around that. a9b2a6683f583811bb3a284e23e4e14089914386

But apparently it didn't quite fix it in Safari.

If I remove the autofocus, its fine (but obviously the field doesn't get focused). 🤔

**Comment by jasonvarga** [comment]
I usually use Safari, except when in heavy Vue development. Safari doesn't have Vue devtools. 😢

**Comment by robdekort** [comment]
Ah gotcha. That's a bummer!

**Comment by jasonvarga** [comment]
Looks like this was fixed in #7794
